tensorflow_io/arrow/python/kernel_tests/arrow_test.py

```diff
@@ -305,6 +308,40 @@ def run_server(num_batches):
 
     server.join()
 
+  @unittest.skipIf(not _have_pyarrow, _pyarrow_requirement_message)
+  def testBoolArrayType(self):
+
+    # NOTE: need to test this seperately because to_pandas fails with
+    # ArrowNotImplementedError: Not implemented type for list in
```

Comment:
Is this a known issue?

Reply:
@xhochy  I'm not sure, I just noticed it with this pr. I'll look into it and make a jira if one doesn't exist already.

Reply:
Found the issue https://issues.apache.org/jira/browse/ARROW-4370

Reply:
@BryanCutler Thanks for digging into this!

Reply:
Could you add the link to the issue in the comment?

Reply:
yup, good idea